src/main/java/edu/harvard/iq/dataverse/pidproviders/PidProviderFactoryBean.java

```diff
@@ -188,6 +188,9 @@ private void loadProviders() {
                         legacy = new FakeDOIProvider("legacy", "legacy", authority, shoulder, identifierGenerationStyle,
                                 dataFilePidFormat, "", "");
                         break;
+                    case "CrossRef":

```

Comment:
I think you should drop this. If you want to support legacy still, this section should be doing like the DataCite legacy section above to read the legacy-style options and feed them into the provider directly (creating a new CrossRefDOIProvider rather than calling a factory method). Again - probably better to drop it.

Reply:
will remove